src/indigo/less/indigo.less

```diff
@@ -16,15 +15,10 @@
 @import "bootstrap/less/navs";
 @import "bootstrap/less/navbar";
 @import "bootstrap/less/breadcrumbs";
-@import "bootstrap/less/pagination";
 @import "bootstrap/less/labels";
-@import "bootstrap/less/badges";
-@import "bootstrap/less/thumbnails";
 @import "bootstrap/less/progress-bars";
-@import "bootstrap/less/media";
 @import "bootstrap/less/list-group";
 @import "bootstrap/less/panels";
-@import "bootstrap/less/responsive-embed";
```

Comment:
Žádné tyto styly nepoužíváme tak vyhozeno. Otázka je asi pouze ten `print` zda se může hodit, ale moc tomu nedávám.

Reply:
Ten se používá kde?

Reply:
?

Reply:
@natocTo

Reply:
No jakže nikde, to je média queries pokud by šel někdo tisknout nějakou stránku v keboole.. to právě ani tak absolutně Neředíně tak netřeba ani tento dwfault myslim